### ISSUE TITLE
Sorry to use the issue tracker again...

### ISSUE THREAD

**Issue by Leonard-The-Wise** [wxWidgets/wxWidgets#22253]
But I sent some more messages yesterday and since I got no reply on email and I wanted to reach @vadz at least this one last time again for a comment and don't know if I got spam filtered or anything like that, then I'll post it here.

Dear Mr. @vadz, Leonardo Silva again, the guy from emails yesterday. 
This is **not** another demand, nor question, nor issue, noir me trying to stretch that one discussion, nor anything like that. 

I just felt like I have to explaining myself to you a bit: that when I did reach you saying all those things about how I felt about our previous interactions, etc., I only want to say that in **any way** I was trying to bully you around or to pressure on you to change yourself and the way you express yourself normally. I just felt that in that time I (we) were having problems communicating with each other, but since our last interaction, I stand my ground saying to you that these issues are now fixed for good.

In other words: this is just me explaining myself to you that I was not trying in any way to bully your person nor anything like that. That's not how I am. I just had a small issue I opened myself to you, but now exerything is fine again ok? So, aside from that small incident, all I wish to you is to please, just keep being yourself and continue to do the wonderful work you are doing for the community, ok? Your product is really fantastic and your volunteer work here during all these years is extraordinary. :)

Those are my final words to you. You do not need to verbally reply here, you may just only close the ticket.

All I wanted here is to have a confirmation that this message DID make it to you.

Goodbye, at least for now.

Yours trully,
Leonardo Silva

**Comment by Leonard-The-Wise** [comment]
Hope you didn't forget me already eheh. Anyway.

**Comment by Leonard-The-Wise** [comment]
In any case I could just remember you. Lol... anyway, sorry, last post.
